scripts/pkg_in_pipe/pkg_in_pipe.py

```diff
@@ -136,20 +136,20 @@ def print_table_line(out, build, link, issues, built_by, prs: list[PullRequest],
             <th scope="row" class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white">
```

Comment:
Why not here too?

Reply:
That's an overlook. The result is even more dense with it, perhaps a bit too much to my taste

Reply:
We can adjust later if that's too much, then.